probatus/feature_elimination/feature_elimination.py

```diff
@@ -292,6 +292,8 @@ def _report_current_results(
         train_metric_std,
         val_metric_mean,
         val_metric_std,
+        features_mean_abs_shap_value,
```

Comment:
Is this a dictionary or a list?

If a list, please make sure that this corresponds well to the current feature list. the feature set might be sorted based on the shap values. Something to double check :)